apps/prairielearn/src/tests/courseEditor.test.ts

```diff
@@ -311,10 +315,17 @@ const testEditData: EditData[] = [
   },
 ];
 
+function getPostInfoFromCopyOption(form) {
+  const option = form.find('select[name="to_course_id"] option[value="1"]');
+  return { csrfToken: option.attr('data-csrf-token'), url: option.attr('data-copy-url') };
+}
```

Comment:
The previous version of the test was using the default action from the form, however the modal for copying dynamically updates the URL and token based on the selected value. This wasn't an issue before because the select only had one course to choose from, so the default matched, but now it doesn't match anymore because the select includes two courses (the destination for testing and the same course).